tests/staticfiles_tests/test_checks.py

```diff
@@ -132,3 +136,57 @@ def test_nonexistent_directories(self):
             # Nonexistent directories are skipped.
             finder = get_finder("django.contrib.staticfiles.finders.FileSystemFinder")
             self.assertEqual(list(finder.list(None)), [])
+
+
+class StoragesCheckTests(SimpleTestCase):
+    @override_settings(STORAGES={})
+    def test_error_empty_storages(self):
+        # "default" and "staticfiles" need to be popped from STORAGES since
+        # UserSettingsHolder has code to maintain backward compatibility.
+        # RemovedInDjango51Warning
+        settings.STORAGES.clear()
+        assert settings.STORAGES == {}
+        errors = check_storages(None)
+        self.assertEqual(
+            errors, [Error(MISSING_STORAGE_SETTING, id="staticfiles.E005")]
+        )
+
+    def test_no_errors_storages_default_value(self):
+        assert sorted(settings.STORAGES.keys()) == ["default", "staticfiles"]
+        errors = check_storages(None)
+        self.assertEqual(errors, [])
+
+    @override_settings(
+        STORAGES={
+            "staticfiles": {
```

Comment:
I'd use `STATICFILES_STORAGE_ALIAS` instead.
```suggestion
            STATICFILES_STORAGE_ALIAS: {
```